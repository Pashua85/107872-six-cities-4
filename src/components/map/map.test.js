import React from 'react';
import Enzyme, {mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Map from './map';
import places from '../../mock/test-offers';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`Map`, () => {
  it(`sould rendred correctly`, () => {
    const wrapper = mount(
        <Map places={places} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
