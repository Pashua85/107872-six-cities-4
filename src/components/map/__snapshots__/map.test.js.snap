// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map sould rendred correctly 1`] = `ReactWrapper {}`;
